src/scss/layout/_cta.scss

```diff
@@ -8,3 +10,40 @@
   grid-template-columns: minmax(min-content, var(--page));
   justify-content: center;
 }
+
+.hire-us-cta {
+  --reverse-y: 'content' 'image';
+  --media-gap: var(--double-gutter);
```

Comment:
```suggestion
  --media-gap: var(--double-gutter);
  --reverse-y: 'content' 'image';
```

Reply:
fixed in https://github.com/oddbird/oddleventy/pull/808/commits/15b0a3f91ab66ebdb1aaf83febbce025309cdbfe